Advice

hellolisaberg opened the pull request.

redrambles commented:
Hey @hellolisaberg !

Apologies for the delay! So this pull request is coming along - there are just a few things we need to address before we're ready to merge this in.

1) Have a look at the files included in your pull request [here](https://github.com/skillcrush/105-practice/pull/861/files). You have two sets of html and css files - but we only need one. You are linking to the `advice/lisa-b-advice.html` file in the main index.html - so please delete the `advice/lisabadvice.html` and the `css/lisabadvice.css` files.

2) You made a change to the shared stylesheet (`css/style.css`) - see [here](https://github.com/skillcrush/105-practice/pull/861/files#diff-6a6a912d0c7b55b537768da778032849L6). Please remove the additional style so that it remains as it was originally.


Once you have deleted the two extra files and removed the change you added to the shared css/style.css file - please add `git add -A` and commit `git commit -m "Your message here"` and push up your advice branch once more `git push origin advice`. This will automatically update this pull request, and then we should be all set to merge in your code.

Please let us know if you have any questions!

Cheers,
Ann :)

hellolisaberg commented:
> Hey @hellolisaberg !
> 
> Apologies for the delay! So this pull request is coming along - there are just a few things we need to address before we're ready to merge this in.
> 
> 1. Have a look at the files included in your pull request [here](https://github.com/skillcrush/105-practice/pull/861/files). You have two sets of html and css files - but we only need one. You are linking to the `advice/lisa-b-advice.html` file in the main index.html - so please delete the `advice/lisabadvice.html` and the `css/lisabadvice.css` files.
> 2. You made a change to the shared stylesheet (`css/style.css`) - see [here](https://github.com/skillcrush/105-practice/pull/861/files#diff-6a6a912d0c7b55b537768da778032849L6). Please remove the additional style so that it remains as it was originally.
> 
> Once you have deleted the two extra files and removed the change you added to the shared css/style.css file - please add `git add -A` and commit `git commit -m "Your message here"` and push up your advice branch once more `git push origin advice`. This will automatically update this pull request, and then we should be all set to merge in your code.
> 
> Please let us know if you have any questions!
> 
> Cheers,
> Ann :)

Hi @redrambles!
I think I got these fixed. I guess I still had my old files in there from when I did the first version of the Git class back in December but I removed them. Also it looks like I did some edits on the main CSS thinking I was working on the CSS for another project- oops!

Thanks!
Lisa

redrambles commented:
Hey @hellolisaberg !

Thanks for making those changes - your code is all merged in! :)

Cheers,
Ann :)
